src/panels/GlobeImageryBrowserPanel/GlobeImageryBrowserPanel.tsx

```diff
@@ -0,0 +1,259 @@
+      refreshGlobeBrowsingNodes();
+    }
+  }, [globeWMS, selectedWMS, refreshGlobeBrowsingNodes]);
+
+  // Sets default selected globe node
+  useEffect(() => {
+    if (!globeBrowsingNodes || selectedGlobe) {
+      return;
+    }
+    // Set Earth as default selected if it is loaded in the scene
+    const selectGlobe = globeBrowsingNodes.identifiers.includes('Earth')
+      ? 'Earth'
+      : globeBrowsingNodes.identifiers[0];
+
+    setSelectedGlobe(selectGlobe);
+  }, [globeBrowsingNodes, selectedGlobe]);
+
+  async function addLayer(cap: Capability, layerType: LayerType) {
+    if (!selectedGlobe) {
+      return;
+    }
+
+    const layerName = makeIdentifier(cap.Name);
+
+    await luaApi?.globebrowsing.addLayer(selectedGlobe, layerType, {
+      Identifier: layerName,
+      Name: cap.Name,
+      FilePath: cap.URL,
+      Enabled: true
+    });
+  }
+
+  async function removeLayer(name: string) {
+    if (!selectedGlobe) {
+      return;
+    }
+
+    const layerName = makeIdentifier(name);
+    for (const layerType of layerTypes) {
+      if (addedLayers[layerType].includes(layerName)) {
+        await luaApi?.globebrowsing.deleteLayer(selectedGlobe, layerType, layerName);
+      }
+    }
+  }
+
+  async function onAddServer(name: string, url: string) {
+    if (!selectedGlobe) {
+      return;
+    }
+    await luaApi?.globebrowsing.loadWMSCapabilities(name, selectedGlobe, url);
+    refreshWMSInfo();
+    setSelectedWMS(name);
+    refreshGlobeBrowsingNodes();
+    close();
+  }
+
+  function removeServerModal() {
+    if (!selectedWMS) {
+      return;
+    }
+    modals.openConfirmModal({
+      title: t('remove-server-modal.title'),
+      children: (
+        <Text>{t('remove-server-modal.description', { serverName: selectedWMS })}</Text>
+      ),
+      labels: {
+        confirm: t('remove-server-modal.confirm'),
+        cancel: t('remove-server-modal.cancel')
+      },
+      onConfirm: async () => {
+        await luaApi?.globebrowsing.removeWMSServer(selectedWMS);
+        refreshWMSInfo();
+      },
+      confirmProps: { color: 'red', variant: 'filled' }
+    });
+  }
+
+  if (!globeBrowsingNodes) {
+    return <LoadingBlocks />;
+  }
+
+  return (
+    <>
+      <AddServerModal opened={opened} close={close} onAddServer={onAddServer} />
+      <Layout>
+        <Layout.FixedSection>
+          <Stack gap={'xs'} mb={'xs'}>
+            <Group gap={'xs'}>
+              <Select
+                value={selectedGlobe}
+                data={[
+                  {
+                    group: '',
+                    items: globeBrowsingNodes.identifiers.slice(
+                      0,
+                      globeBrowsingNodes.firstIndexWithoutUrl
+                    )
+                  },
+                  {
+                    group: t('select-globe.no-server-group'),
+                    items: globeBrowsingNodes.identifiers.slice(
+                      globeBrowsingNodes.firstIndexWithoutUrl
+                    )
```

Comment:
For readability, maybe create two variables for these two items lists? It would make this much easier to read and understand if they were variables such as `globesWithServers`